### ISSUE TITLE
A_Expr printer does not handle missing lexpr

### ISSUE THREAD

**Issue by aiham** [lelit/pglast#53]
When prefixing an expression with a minus sign the query is correctly parsed but the A_Expr printer fails. It works if the minus sign has a value to the left of it.

```py
>>> from pglast import printers, Node, parse_sql
>>> from pglast.printer import RawStream
>>>
>>>
>>> RawStream()(Node(parse_sql('SELECT 0-(1+1)')))
'SELECT 0 - ((1 + 1))'
>>>
>>>
>>> RawStream()(Node(parse_sql('SELECT -(1+1)')))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printer.py", line 251, in __call__
    self.print_node(statement)
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printer.py", line 380, in print_node
    printer(node, self)
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printers/dml.py", line 701, in raw_stmt
    output.print_node(node.stmt)
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printer.py", line 380, in print_node
    printer(node, self)
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printers/dml.py", line 777, in select_stmt
    output.print_list(node.targetList)
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printer.py", line 437, in print_list
    is_symbol=is_symbol)
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printer.py", line 401, in _print_items
    self.print_node(item, is_name=are_names, is_symbol=is_symbol and idx == last)
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printer.py", line 380, in print_node
    printer(node, self)
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printers/dml.py", line 707, in res_target
    output.print_node(node.val)
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printer.py", line 380, in print_node
    printer(node, self)
  File "/Users/aiham/Library/Caches/pypoetry/virtualenvs/aiham-parser-UHj-Q3Ed-py3.7/lib/python3.7/site-packages/pglast/printers/dml.py", line 33, in a_expr
    if node.lexpr.node_tag == 'A_Expr':
AttributeError: 'Missing' object has no attribute 'node_tag'
```

This is a valid expression:

```sql
postgres=# SELECT -(1+1);
 ?column?
----------
       -2
(1 row)
```

The data structure for `SELECT -(1+1)` is:

```json
[
 {
  "RawStmt": {
   "stmt": {
    "SelectStmt": {
     "targetList": [
      {
       "ResTarget": {
        "val": {
         "A_Expr": {
          "kind": 0,
          "name": [
           {
            "String": {
             "str": "-"
            }
           }
          ],
          "rexpr": {
           "A_Expr": {
            "kind": 0,
            "name": [
             {
              "String": {
               "str": "+"
              }
             }
            ],
            "lexpr": {
             "A_Const": {
              "val": {
               "Integer": {
                "ival": 1
               }
              },
              "location": 9
             }
            },
            "rexpr": {
             "A_Const": {
              "val": {
               "Integer": {
                "ival": 1
               }
              },
              "location": 11
             }
            },
            "location": 10
           }
          },
          "location": 7
         }
        },
        "location": 7
       }
      }
     ],
     "op": 0
    }
   }
  }
 }
]
```

This is where it fails:

https://github.com/lelit/pglast/blob/ad450deb09e01645179553f489211833e8a885ed/pglast/printers/dml.py#L33

The printer heavily assumes lexpr always exists so I'm not sure what the best way to fix it is.

**Comment by lelit** [comment]
Thanks for the report. At first, I'd try to wrap the handling of the the lexpr up to the separator space in an ``if`` like:
```python
        with output.expression():
            if node.lexpr is not Missing:
                if node.lexpr.node_tag == 'A_Expr':
                    with output.expression():
                        output.print_node(node.lexpr)
                else:
                    output.print_node(node.lexpr)
                output.write(' ')
            if isinstance(node.name, List) and len(node.name) > 1:
               ...
```

I cannot focus on this right now, maybe you can set up a test and give the above a try?

**Comment by aiham** [comment]
Thanks, submitted #54

**Comment by lelit** [comment]
Thank you, released in v1.11.
